### DISCUSSION TITLE
With external composition Federation 2 support does it support the @authenticated directive

### DISCUSSION THREAD

**Discussion by ArtSecureId** [graphql-hive/console#3163] [Feature request]
Just need a little bit of infomation regarding the new directives like @authenticated in Federation v2.
When it mention in the Apollo Router documentation that we need an Entreprise License with GraphOS / Apollo Router for some directives, since we use Hive instead, does it mean that it needs to be developed within Hive otherwise no support is done for this directive ?

Thanks,

**Comment by n1ru4l** [comment]
Hey, you are correct, `@authenticated` is a feature of GraphOS/ApolloRouter and cannot be offered by us for Apollo Router.

We recommend you keep an eye on https://the-guild.dev/graphql/gateway to learn more about a new custom OSS router that can support `@authenticated` without violating the license of Apollo Router.

Support for `@authenticated` in conductor is not yet planned, but we can prioritize it based on the urgency of needs. Feel free to get in contact via the form on the website.
